packages/neutrino/bin/start.js

```diff
@@ -1,31 +1,50 @@
 const { start } = require('../src');
 const ora = require('ora');
+const { lookup } = require('dns');
+const { hostname } = require('os');
+const Future = require('fluture');
+
+const whenIpReady = Future.node((done) => {
+  const host = hostname();
+  lookup(host, done);
+});
 
 module.exports = (middleware, options) => {
   const spinner = ora('Building project').start();
 
-  return start(middleware, options)
-    .fork((errors) => {
-      spinner.fail('Building project failed');
-      errors.forEach(err => console.error(err));
-      process.exit(1);
-    }, (compiler) => { // eslint-disable-line consistent-return
-      if (!compiler.options.devServer) {
-        return spinner.succeed('Build completed');
-      }
+  return whenIpReady
+    .chain(ip => Future.of(process.env.HOST = process.env.HOST || ip))
```

Comment:
I decided to use `process.env.HOST` as a shared variable between `start.js ` and the middleware. Thought it is better than to create a new global variable. We keep the host in `HOST` anyway so I update it if it is not set from the CLI. 
I had question here that was left without an answer. Do we ever need to define the host from the CLI?